lib/asciidoctor/js/opal_ext/file.rb

```diff
@@ -139,6 +141,64 @@ def self.read(path)
     end
   end
 
+  # Monkey-patch to keep path unchanged on absolute path:
+  # * URLs starting with file://, http:// or https://
+  # * Windows path starting with a drive letter
+  def self.expand_path(path, basedir = nil)
+    if path.start_with?('file://') || path.start_with?('http://') || path.start_with?('https://')
+      return path
+    end
+    if WindowsRootRx.match? (path)
+      return path
+    end
+    # FIXME: The code is duplicated, how to call "super" ?
```

Comment:
@mojavelinux I tried to use your "technique" but didn't succeed 🤕 

```js
const thisExtension = this,
    Html5Converter = Opal.klass(asciidoctor.$$const.Converter, null, 'Html5Converter', () => {})

Opal.alias(Html5Converter, 'super_inline_anchor', 'inline_anchor')
```

Reply:
You need to first alias expand_path to super_expand_path. Then you can call the alias method.